packages/xpetra/sup/Utils/Xpetra_TripleMatrixMultiply.hpp

```diff
@@ -202,6 +226,37 @@ namespace Xpetra {
 #else
         throw(Xpetra::Exceptions::RuntimeError("Xpetra must be compiled with Tpetra."));
 #endif
+        if (call_FillComplete_on_result && !haveMultiplyDoFillComplete) {
+          RCP<Teuchos::ParameterList> fillParams = rcp(new Teuchos::ParameterList());
+          fillParams->set("Optimize Storage", doOptimizeStorage);
+          Ac.fillComplete((transposeP) ? P.getRangeMap() : P.getDomainMap(),
```

Comment:
I don't understand why new code is here. This is a change in previous behavior. Was `Ac` not fill-completed before? Is that a fix for something?

Reply:
It's a copy-paste to match what the general implementation has. This was missing in the specialization, so I simply added it.

Reply:
Hmm, this would mean that there was a bug here and we never fill-completed the matrix?

Reply:
I guess it's possible. This was probably dependent on the other parameters that were set, and since the triple product wasn't used much, we didn't notice.